react-18/client/hooks/index.ts

```diff
@@ -8,13 +7,18 @@ import * as React from 'react';
  * @param {function} onBinder function to be executed when the event is fired
  * @param {string} event the event to be listened, by default is event change
  */
-export /*bundle*/
-function useBinder(objects: any[], onBinder: () => void, event: string = 'change'): void {
-    React.useEffect((): (() => void) => {
-        objects.forEach((object: any): void => {
-            if (!object) throw new Error(`object is not valid in useBinder ${object}`);
-            object.on(event, onBinder);
+ export /*bundle*/
+ function useBinder(objects: any[], onBinder: () => void, events: string | string[] = 'change'): void {
+   const bindEvents : string[] = typeof events === 'string' ? [events] : events;
+    React.useEffect(() => {
+        objects.forEach(object => {
+            if(!object?.bind) return
```

Comment:
The current implementation of the `useBinder` hook only provides support for `bind` and `unbind` event methods. However, our event handling convention utilizes `on` and `off` methods. We need to update the hook logic to align with this convention. Please revise the hook to include support for `on` and `off` methods for event listening and unbinding, respectively. This change is necessary to ensure compatibility with our existing event management system.

Reply:
Ready 🤓☝